_docs/_latest/setup/upgrade.md

```diff
@@ -866,6 +866,71 @@ to the Client ID of your service account.
 {% endcapture %}
 {% include site/zippy/item.html title="Upgrading 2.8.0 to 2.9.0" content=upgrading_2_8_0_to_2_9_0 uid=10 %}
 
+{% capture upgrading_2_9_0_to_2_10_0 %}
+
+1. Open cloud shell when you are in the Forseti project on GCP.
```

Comment:
You need to add a section for role updates:

https://github.com/GoogleCloudPlatform/forseti-security/pull/2395